examples/advanced/tidb-ng-monitoring.yaml

```diff
@@ -0,0 +1,182 @@
+  configUpdateStrategy: RollingUpdate
+
+  ## Persistent volume reclaim policy applied to the PV consumed, default to `Retain`.
+  ## Note that the reclaim policy Recycle may not be supported by some storage types, e.g. local.
+  ## Ref: https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/
+  pvReclaimPolicy: Retain
+
+  ## ImagePullPolicy of Pods
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/overview/#container-images
+  # imagePullPolicy: IfNotPresent
+
+  ## If private registry is used, imagePullSecrets may be set
+  ## You can also set this in service account
+  ## Ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # imagePullSecrets:
+  # - name: secretName
+
+  ## Specify pod priorities of pods in TidbNGMonitoring, default to empty.
+  ## Can be overwritten by component settings.
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  # priorityClassName: system-cluster-critical
+
+  ## Use the node network namespace, default to false
+  ## Ref: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#host-namespaces
+  # hostNetwork: false
+
+  ## if true, this ng monitoring cluster is paused and will not be synced by the controller
+  # paused: false
+
+  ## SchedulerName of pods.
+  ## If specified, the pods will be scheduled by the specified scheduler.
+  ## Can be overwritten by component settings.
+  # schedulerName: default-scheduler
+
+  ###########################
+  # Components              #
+  ###########################
+
+  ngMonitoring:
+    ## describes the compute resource requirements and limits.
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+    requests:
+    #   cpu: 1000m
+    #   memory: 1Gi
+      storage: 10Gi
+    # limits:
+    #   cpu: 2000m
+    #   memory: 2Gi
+
+    ## version of ng-monitoring
+    version: nightly
+    
+    ## Base image of the component
+    baseImage: pingcap/ng-monitoring
+
+    ## The storageClassName of the persistent volume for data storage.
+    # storageClassName: standard
+
+    ## ng-monitoring configuration
+    # config: |
+    #   [log]
+    #     level = "DEBUG"
+
+    ## Annotations of ng monitoring pods
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    #   topology.kubernetes.io/region: some-region
+
+    ## Labels of ng monitoring pods
+    # labels: {}
+
+    ## List of environment variables to set in the container
+    ## Note that the following env names cannot be used and will be overwritten by TiDB Operator builtin envs
+    ##   - NAMESPACE
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
```

Comment:
these two lines can be dropped?

Reply:
Please also check if the others are set for ngm.

Reply:
done